vendor/engines/secure_rooms/app/views/secure_rooms/shared/_secure_rooms_dashboard.html.haml

```diff
@@ -0,0 +1,27 @@
+-# TODO: Styling needs work
```

Comment:
I think this looks good enough for a V1. We can adjust the styling as we do the 📺  version.